.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,8 +16,8 @@ Check these items off in the GitHub PR UI as you complete them.
 
 <!-- 
 Make sure to remove any '.' characters from the branch name 
-Example: ks.my-branch --> ksmy-branch
--->
+Example: ks.my-branch --> 
```

Comment:
Oh I see what it's doing, it thinks the `-->` between the branch names is the end of the comment! I think if you update it to be one hyphen in the arrow you should be good.

```suggestion
Example: ks.my-branch -> ksmy-branch
-->
```